theme/components/sections/header.html.twig

```diff
@@ -313,3 +313,32 @@
         }
     }
 {% endschema %}
+
+{% editor %}
+{
+    "name": "Header",
```

Comment:
Just a heads up, there is a bug where the platform cannot read section controls when the section lives in the layout, these controls are currently not shown